ccd-sample-data/src/main/java/uk/gov/hmcts/cmc/ccd/sample/data/SampleData.java

```diff
@@ -888,4 +888,40 @@ public static CCDCase withPaperResponseFromScannedDoc() {
                         .build()).build()
         );
     }
+
+    public static CCDRespondent withRespondentPartyNameAsNull() {
+        CCDAddress ccdAddress = getCCDAddress();
+        return CCDRespondent.builder()
+            .partyName(null)
+            .claimantProvidedDetail(
+                CCDParty.builder()
+                    .type(INDIVIDUAL)
+                    .primaryAddress(ccdAddress)
+                    .dateOfBirth(LocalDate.of(1950, 1, 1))
+                    .correspondenceAddress(ccdAddress)
+                    .build())
+            .partyDetail(
+                CCDParty.builder()
+                    .primaryAddress(ccdAddress)
+                    .correspondenceAddress(ccdAddress)
+                    .build())
+            .claimantProvidedPartyName("Mary Richards")
+            .claimantProvidedRepresentativeOrganisationAddress(ccdAddress)
+            .claimantProvidedRepresentativeOrganisationName("My Org")
+            .claimantProvidedRepresentativeOrganisationPhone("07987654321")
+            .claimantProvidedRepresentativeOrganisationPhone("my@email.com")
```

Comment:
It is best to define these string literals in a within a constant, it would even be worth creating a constant class which can be called from.

Reply:
Solved. Have a look when you can